src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -31,26 +30,35 @@ public class ExpressionEvaluationTests : PowerFxTest
         // - parameters: to define the parameters for the test case
         //               receives an 'object' if defined in updatePfxConfig
         // - configureEngine: to configure the engine
+        // - runtimeConfigTypes: set of objects to add to RuntimeConfig
+        // - setVariables: set variables in the symbol values
         // Each of these actions are executed in order (init, update, param, configure)
-        internal static Dictionary<string, (Func<PowerFxConfig, PowerFxConfig> initPfxConfig, Func<PowerFxConfig, object> updatePfxConfig, Func<object, RecordValue> parameters, Action<RecalcEngine, bool> configureEngine)> SetupHandlers = new ()
+        internal static Dictionary<string, 
+            (Func<PowerFxConfig, PowerFxConfig> initPfxConfig, 
+             Func<PowerFxConfig, SymbolTable, object> updatePfxConfig, 
+             Func<object, RecordValue> parameters, 
+             Action<RecalcEngine, bool> configureEngine,
+             Action<RuntimeConfig> runtimeConfig,
+             Action<ReadOnlySymbolValues, ISymbolSlot[]> setVariables)> SetupHandlers = new ()

```

Comment:
do mutation tests, like collect.txt, handle this?

Reply:
AFAIK, yes